### DISCUSSION TITLE
Webhook rule defaults to developer email when not provided

### DISCUSSION THREAD

**Discussion by misimpso** [codatio/codat-docs#261] [Q&A] [answered]
As part of our integration creation flow, we're programatically creating a webhook rule to know when to start ingesting of the data. The webhook we're creating only includes a webhook url as its notifier, and it doesn't include an email. It looks to be defaulting to the developer email tied to our codat account and is spamming that account with webhook events. This just started happening last friday and since then we've received over 300 emails.

I don't see anything in the documentation for the webhook or rule creation that describes this behavior, and our rule creation code on our end hasn't changed. How do we disable the account email from being used as the notification email for these rules?

**Comment by mcclowes** [comment, marked as answer]
Hi @misimpso, let me investigate this and get back to you!

**Reply by misimpso** [reply]
Hey @mcclowes, I was able to speak someone from Codat support. Looks like the emails we were receiving were from old alerts that were configured with an email. We have since stopped attaching an email to these alerts, and with confirmation from Codat support, confirmed that these alerts were not generating emails. Sorry for the false alarm

**Reply by pzaichkina** [reply]
Hi @misimpso, thank you for letting us know - glad to hear your issue was resolved!
